### PR TITLE
Prepare 3.5.1 release.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 3.5.1 (2015-10-28)
+
+* (mvz) Make Attribute respect suppressing comments
```
Comment (mvz) [inline CHANGELOG.md]: Perhaps add other changes as well?

* Changed parser dependency
* Allow matches in `reek_of` for message, lines, context and source.
* Improved documentation
* Add logo!
* Deprecate AppConfiguration.from_map in favor of AppConfiguration.from_hash.

Comment (mvz) [inline CHANGELOG.md]: Hm. Sounds more like a minor release :smile:.

Comment (troessner) [inline CHANGELOG.md]: >> Improved documentation
Add logo!

But those 2 things, as awesome as they, are non relevant for a user from a versioning point of view, aren't they?

I just added

>> Adjust parser dependency to allow versions 2.2.3+ (and even 2.3+)
Allow matches in reek_of for message, lines, context and source.
Deprecate AppConfiguration.from_map in favor of AppConfiguration.from_hash.

to the changelog and updated this PR to be a minor release.

**Comment by troessner** [comment]
Closing this one in favor of #758

**Pull request closed without merging**